tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -320,55 +319,69 @@ void read_args(int argc, char **argv){
 
 
 //print matrix : row major, i = line, j = column
-void print_host2D(float * tab) {
-  for (size_t i = 0; i < M; ++i){
-    for(size_t j = 0; j < N; ++j){
-      std::cout << tab[i*N+j] << " " ;
+void print_host2D(float * tab)
```

Comment:
Apologies. Still working on my group development skills/etiquette. Sometimes VS Auto Formatting kicks in (I like leaving it on), sometimes my fingers go overboard. I'll revert.